src/pages/movie/[id].js

```diff
@@ -0,0 +1,144 @@
+import Head from 'next/head';
+import {useRouter} from 'next/router';
+import styled from 'styled-components';
+
+import Layout from '../../components/Layout';
+import useFetch from '../../hooks/useFetch';
+import useStore from '../../hooks/useStore';
+
+const Wrapper = styled.section`
+	padding: 24px;
+`;
+const Picture = styled.img`
+	max-width: 185px;
+	border-radius: 16px 16px 16px 16px;
+	box-shadow: 1px 1px 10px 5px rgba(0, 0, 0, 0.2);
+`;
+
+const Hl2 = styled.h2`
+	padding: 16px 0 0 0;
+	color: white;
+	overflow-wrap: break-word;
+	font-size: 36px;
+	font-style: normal;
+	font-weight: 600;
+	& span {
+		font-weight: 400;
+	}
+`;
+
+const Hl3 = styled.h3`
+	padding: 0 8px 8px 0;
+	color: white;
+	overflow-wrap: break-word;
+	font-size: 20px;
+	font-style: italic;
+	font-weight: 400;
+`;
+
+const Hl4 = styled.h4`
+	padding: 24px 8px 0 0;
+	color: white;
+	overflow-wrap: break-word;
+	font-size: 18px;
+	font-style: normal;
+	font-weight: 600;
+	line-height: 1.3;
+	& span {
+		font-style: normal;
+		font-weight: 200;
+	}
+`;
+
+const Article = styled.article`
+	display: flex;
+	padding: 24px 0;
+`;
+
+const Paragraph = styled.p`
+	padding: 0 32px;
+	color: white;
+	line-height: 1.5;
+`;
+
+export default function Movie() {
+	// playtime helper function
+	function toHoursAndMinutes(totalMinutes) {
+		const minutes = totalMinutes % 60;
+		const hours = Math.floor(totalMinutes / 60);
+
+		return `${hours}h ${padTo2Digits(minutes)}m`;
+	}
+	function padTo2Digits(num) {
+		return num.toString().padStart(2, '0');
+	}
+
+	const router = useRouter();
+	const id = router.query.id;
+
+	const API_KEY = process.env.API_KEY;
+	const SINGLE_MOVIE = `https://api.themoviedb.org/3/movie/${id}?api_key=${API_KEY}&language=en-US`;
+	const {loading, error, data: objData} = useFetch(SINGLE_MOVIE);
+	const objConfiguration = useStore(state => state.objConfiguration);
+	const imagesBaseUrl = objConfiguration.secure_base_url + objConfiguration.poster_sizes[2];
+	const backdropImageUrl = objConfiguration.secure_base_url + objConfiguration.backdrop_sizes[1];
+
+	return (
+		<Layout>
+			<Head>
+				<title>Movies</title>
+			</Head>
+			{loading && <p>loading...</p>}
+			{error && <p>The content could not be loaded. Try again later.</p>}
+			{objData && (
+				<div>
+					<Wrapper
```

Comment:
Inline styling again.

Reply:
As mentioned above, haven't found a JS solution to do this otherwise.